website/project/model.py

```diff
@@ -148,13 +147,31 @@ class MetaData(GuidStoredObject):
     date_created = fields.DateTimeField(auto_now_add=datetime.datetime.utcnow)
     date_modified = fields.DateTimeField(auto_now=datetime.datetime.utcnow)
 
+class Commentable(object):
```

Comment:
Also, this class could use a brief docstring so that devs know when to use it. Something simple like `"""Abstract class that defines the interface for models that can have comments attached to them."""`